### PR TITLE
Rapid breach response log4j pack

### PR DESCRIPTION
**Pull request by melamedbn**
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A new pack for handling the new vulnerability cve-2021-44228 in log4j library.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CVE_2021_44228/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+This pack is part of the Rapid Breach Response pack.
+
+**Critical RCE Vulnerability: log4j - CVE-2021-44228** refers to a 0-day exploit in the popular Java logging library log4j2.
+
+On Dec. 9, 2021, a remote code execution (RCE) vulnerability in Apache log4j 2 was identified being exploited in the wild. Public proof of concept (PoC) code was released and subsequent investigation revealed that exploitation was incredibly easy to perform.
+
+**Affected Version**
+
+Apache Log4j 2.x <= 2.15.0-rc1
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/README.md:9]: The arrow should be in the opposite direction? @melamedbn

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| EDLDomainBlocklist | The EDL domain blocklist name. |  | Optional |
+| BlockIndicatorsAutomatically | Whether to block the indicators automatically or not. | True | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![CVE-2021-44228 - Log4j RCE](Insert the link to your image here)
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:78]: No image? @melamedbn

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+**Critical RCE Vulnerability: log4j - CVE-2021-44228**
+
+On Dec. 9, 2021, a remote code execution (RCE) vulnerability in Apache log4j 2 was identified being exploited in the wild. Public proof of concept (PoC) code was released and subsequent investigation revealed that exploitation was incredibly easy to perform.
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:3]: ...was identified **_as_** being exploited in the wild.

Note: that this sentence is used in other areas of this pack

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 4]
```diff
@@ -0,0 +1,2437 @@
...
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: d1589f39-7ac3-43d6-8580-95ff0dbc1021
+    type: condition
+    task:
+      id: d1589f39-7ac3-43d6-8580-95ff0dbc1021
+      version: -1
+      name: Is QRadar Enabled?
+      description: Check if Splunk instance is enabled.
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:817]: @melamedbn - Task name is for Qradar, but in the description, you writing about Splunk.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 5]
```diff
@@ -0,0 +1,2437 @@
...
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: QRadar_v2
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:847]: @melamedbn - you don't prefer to use "IsIntegrationAvailble"?

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 6]
```diff
@@ -0,0 +1,2437 @@
...
+      script: '|||xdr-run-script-execute-commands'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "34"
+    scriptarguments:
+      commands:
+        simple: sudo egrep -i -r '\$\{jndi:(ldap[s]?|rmi|dns):/[^\n]+' /var/log
```
Comment (ssokolovich) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:1090]: Just to be sure - grep or **e**grep?

**Review by ssokolovich — COMMENTED** [review]
@melamedbn - please check my comment. 
Don't forget there PB image in the ReadMe file

File: Packs/CVE_2021_44228/README.md (hunk 1 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/README.md]: Nope, seems good. also present in Unit42 blog.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md (hunk 2 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:78]: great!! fixed

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md (hunk 3 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:3]: For the grammar, I'm keeping it the same as in Unit42 blog.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 4 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:817]: fixed

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 5 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:847]: No, I prefer not to break the playbook flow now.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 6 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:1090]: egrep

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**